### ISSUE TITLE
Does not test common edge cases for integer inputs

### ISSUE THREAD

**Issue by moodysalem** [crytic/echidna#521]
From what I can tell, all inputs are randomly generated and none are hard coded. Here our fuzz test never fails, even though I expect it to when it passes `type(uint256).max` to `storeSqrt`.

https://github.com/Uniswap/uniswap-lib/pull/16/files#diff-98de4e9ca9e8aa86dc81424627940ec3a0f3d6278f3878e8f76f2d57026d3eceR27

If the input is a `uint256`, it should probably always test `uint256(-1)`, `uint248(-1)`, ..., `uint8(-1)`, `0`

If the user is interested in verifying overflow/rounding behavior with many different inputs, it seems there is always significant value in testing these inputs specifically

**Comment by ggrieco-tob** [comment]
It should be using (at least some of) these constants. Let me check something..

**Comment by ggrieco-tob** [comment]
It seems that this feature was working some time ago, but a refactored removed it. I re-added it in #523 . Can you verify if it works better?

**Comment by moodysalem** [comment]
we dont build from source, could you just add a unit test for this? should be pretty easy to verify that the inputs include the max uints/ints

**Comment by ggrieco-tob** [comment]
Sure. This works in fact (I will add it as unit test in the PR later this week):

```solidity
contract C {
    uint internal x;
    function set(uint _x) public {
        x = _x;
    }

    function echidna_test() public returns (bool) {
        return (x+1 != 0);

    }
}
```

We can provide you a pre-compile binary (or docker) of this branch, if you need.

**Comment by moodysalem** [comment]
> We can provide you a pre-compile binary (or docker) of this branch, if you need.

Any way you could push a patch release? If not, a URL to download the binary would work (since we download and call this in a github action) or I could also commit it to the repo, though I'd probably just wait for an official release

Also we are waiting on a release of slither to fix https://github.com/crytic/slither/issues/641

**Comment by ggrieco-tob** [comment]
> Any way you could push a patch release? If not, a URL to download the binary would work (since we download and call this in a github action) or I could also commit it to the repo, though I'd probably just wait for an official release

We could use the artifacts compiled in the Github CI. Let me check that.

> Also we are waiting on a release of slither to fix crytic/slither#641

Slither is quite easy to install, if you have python3/pip: `python3 setup.py install --user` should be enough (you can clean previous installations to make sure the new version is used).

**Comment by ggrieco-tob** [comment]
Fixed in #523
